### ISSUE TITLE
two images per row?

### ISSUE THREAD

**Issue by valonia14** [JoomGallery/BootOne#14]
Hi, I have installed this but there is only one image per row for smaller screens (tablets, mobile..). Is it possible to edit that so that there was two images per row in some screen sizes?

**Comment by Chraneco** [comment]
Yes, for that you have to either override the template files to use different bootstrap column classes or override the Bootstrap media queries in your CSS in order to achieve different collapsing behavior.
